assets/timber.scss.liquid

```diff
@@ -1190,6 +1190,79 @@ th, td {
   border: 1px solid $colorBorder;
 }
 
+/*============================================================================
+  Responsive tables, defined with .table--responsive on table element.
+  Only defined for IE9+
+==============================================================================*/
+html:not(.lt-ie9) {
+  .table--responsive {
```

Comment:
Honestly I thought it would be totally busted in IE8, even with respond.js. [This demo](http://demo.martinpesout.cz/responsive-tables/) proves that wrong. @stevebosworth, can you get a demo shop up with this PR and remove the `:not(.lt-ie9)` - it may work perfectly already :)